go.mod

```diff
@@ -307,6 +306,7 @@ require (
 	github.com/goccy/go-json v0.10.2 // indirect
 	github.com/godror/knownpb v0.1.0 // indirect
 	github.com/golang-jwt/jwt/v4 v4.5.0 // indirect
```

Comment:
Although the changes look good, I suggest that since we have `github.com/golang-jwt/jwt/v4` as an indirect dependency, why not use the function from there as the code is essentially the same and there is no overhead as far as I know. It is just that `github.com/golang-jwt/jwt/v4` becomes a direct dependency.

Reply:
But it is included, right? Indirect are dependencies imported by direct dependencies. So, they are already on the build list. As `github.com/golang-jwt/jwt/v4` is already in the build list, I suggested let we make it a direct dependency instead (not v5, but v4).

What do you think?